### PR TITLE
Fix minor typos in STC docs.

### PR DESCRIPTION
**Pull request by Blake-Madden**

### CODE REVIEW

File: src/stc/stc.interface.h.in [hunk 1]
```diff
@@ -1087,7 +1087,7 @@ public:
 
     @c wxEVT_STC_INDICATOR_RELEASE
 
-    - Generated when a click over a indicator has been released.
+    - Generated when a click-over indicator has been released.
```
Comment (vadz) [inline src/stc/stc.interface.h.in:1090]: Sorry, what does "a click-over indicator" actually mean? Are you sure the dash here is really needed? I.e. isn't this supposed to be a "click (== mouse button press) over an indicator" instead?

**Review by vadz — COMMENTED** [review]
Thanks for the fixes, but I'm not sure about one of them.

Could anyone please check if I'm reading the intended meaning here correctly?

File: src/stc/stc.interface.h.in (hunk 1 above)
Comment (Blake-Madden) [reply src/stc/stc.interface.h.in]: Oh, I misread that, sorry. I just updated the PR to fix that.

**Comment by vadz** [comment]
Thanks for the update!

**Pull request merged**